packages/task/src/browser/quick-open-task.ts

```diff
@@ -185,3 +236,32 @@ export class TaskAttachQuickOpenItem extends QuickOpenItem {
         return true;
     }
 }
+export class TaskConfigureQuickOpenItem extends QuickOpenGroupItem {
+
+    constructor(
+        protected readonly task: TaskConfiguration,
+        protected readonly taskService: TaskService
+    ) {
+        super();
+    }
+
+    getLabel(): string {
+        return `${this.task._source}: ${this.task.label}`;
+    }
+
+    getDescription(): string {
+        if (this.task._scope) {
+            return new URI(this.task._scope).path.toString();
```

Comment:
use `LabelProvider.getLongName`  to get human readable representation of a URI relative the workspace

Reply:
Done